### DISCUSSION TITLE
Next Release?

### DISCUSSION THREAD

**Discussion by ubuntudroid** [GitLiveApp/firebase-kotlin-sdk#407] [General]
I hate to be that guy, but ...

There have been quite a few changes on master since the last release, including critical bugfixes like proper `Timestamp.toMilliseconds()` calculation (as used in the supplied Timestamp serializers - the previous version was unfortunately plain wrong, causing really weird issues on our clients 🙈) and many improvements to the API surface 🤩.

Quite a nice changelog I'd say.

You probably know where this is going: do you plan on releasing the next version of this library anytime soon? 🙏  Pretty please? 😁

**Comment by walkingbrad** [comment]
I've been that guy before too 😜 and also am waiting for a fix I contributed: instantiating products with specific firebase apps.

**Comment by nbransby** [comment]
done!

**Reply by ubuntudroid** [reply]
Awesome, many thanks! 🙏 🚀

**Comment by ubuntudroid** [comment]
@nbransby Looks like something didn't work with the upload to maven. Getting 404 for 1.8.2: https://central.sonatype.com/artifact/dev.gitlive/firebase-auth/1.8.2?smo=true

**Comment by nbransby** [comment]
@michaelprichardson

**Comment by nbransby** [comment]
@ubuntudroid Sorry about that, please try 1.9.0
